modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/upgrade/v10_20_0/test/ObjectFieldUpgradeProcessTest.java

```diff
@@ -0,0 +1,134 @@
+
+/**
+ * @author Jhosseph Gonzalez
+ */
+@RunWith(Arquillian.class)
+public class ObjectFieldUpgradeProcessTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Test
+	public void testUpgrade() throws Exception {
+		ObjectDefinition objectDefinition =
+			ObjectDefinitionTestUtil.publishObjectDefinition(
+				ObjectDefinitionTestUtil.getRandomName(),
+				Collections.singletonList(
+					new TextObjectFieldBuilder(
+					).labelMap(
+						LocalizedMapUtil.getLocalizedMap(
+							RandomTestUtil.randomString())
+					).name(
+						"textObjectField"
+					).build()),
+				ObjectDefinitionConstants.SCOPE_COMPANY,
+				TestPropsValues.getUserId());
+
+		ObjectField displayDate = _objectFieldLocalService.fetchObjectField(
+			objectDefinition.getObjectDefinitionId(), "displayDate");
+		ObjectField expirationDate = _objectFieldLocalService.fetchObjectField(
+			objectDefinition.getObjectDefinitionId(), "expirationDate");
+		ObjectField reviewDate = _objectFieldLocalService.fetchObjectField(
+			objectDefinition.getObjectDefinitionId(), "reviewDate");
+
+		Assert.assertNull("displayDate", displayDate.getName());
+		Assert.assertNull("expirationDate", expirationDate.getName());
+		Assert.assertNull("reviewDate", reviewDate.getName());
+
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+				_CLASS_NAME, LoggerTestUtil.OFF)) {
+
+			UpgradeProcess upgradeProcess = UpgradeTestUtil.getUpgradeStep(
+				_upgradeStepRegistrator, _CLASS_NAME);
+
+			upgradeProcess.upgrade();
+
+			_multiVMPool.clear();
+		}
+
+		displayDate = _objectFieldLocalService.getObjectField(
+			objectDefinition.getObjectDefinitionId(), "displayDate");
+		expirationDate = _objectFieldLocalService.getObjectField(
+			objectDefinition.getObjectDefinitionId(), "expirationDate");
+		reviewDate = _objectFieldLocalService.getObjectField(
+			objectDefinition.getObjectDefinitionId(), "reviewDate");
+
+		Assert.assertEquals("displayDate", displayDate.getName());
+		Assert.assertEquals("expirationDate", expirationDate.getName());
+		Assert.assertEquals("reviewDate", reviewDate.getName());
+
+		ObjectDefinition userObjectDefinition =
+			_objectDefinitionLocalService.getObjectDefinition(
+				TestPropsValues.getCompanyId(), "User");
+
+		displayDate = _objectFieldLocalService.fetchObjectField(
+			userObjectDefinition.getObjectDefinitionId(), "displayDate");
+		expirationDate = _objectFieldLocalService.fetchObjectField(
+			userObjectDefinition.getObjectDefinitionId(), "expirationDate");
+		reviewDate = _objectFieldLocalService.fetchObjectField(
+			userObjectDefinition.getObjectDefinitionId(), "reviewDate");
+
+		Assert.assertNull("displayDate", displayDate.getName());
+		Assert.assertNull("expirationDate", expirationDate.getName());
+		Assert.assertNull("reviewDate", reviewDate.getName());
```

Comment:
Can you do the Assert before so we can guarantee it was null before and after the upgrade, please?